scripts/createEvent.js

```diff
@@ -0,0 +1,36 @@
+const Web3 = require('web3')
+const storageJSON = require('../build/contracts/Storage.json')
+const accounts = require('../accounts.json')
+
+const createEvent = () =>
+  new Promise((resolve, reject) => {
+    const web3 = new Web3(
+      new Web3.providers.WebsocketProvider('ws://localhost:8545')
+    )
+
+    const testKey = web3.utils.fromAscii('testKey')
+
+    const contract = new web3.eth.Contract(
+      storageJSON.abi,
+      storageJSON.networks['123'].address
+    )
+
+    contract.methods
+      .registerData(testKey, 'qm123')
+      .send({ from: accounts[0] }, (error, res) => {
+        if (error) reject(error)
+        resolve(res)
+      })
+  })
+
+const start = async () => {
+  try {
```

Comment:
should eventually throw error, not just console log right?
so we can test

Reply:
Ya, probably. Wondering if we do that now or when we add a test for this.